.github/workflows/codeql.yml

```
@@ -0,0 +1,80 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ 'master' ]
+  schedule:
+    - cron: '14 22 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
+    timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp', 'python' ]
```

We don't have any Python code, so could we speed up the execution the workflow by removing 'Python'?

---

There's a python file [here](https://github.com/sqlitebrowser/sqlitebrowser/blob/2179c5de422e96a56caca30940467bccab49b127/tests/createtestdb.py), but feel free to remove the analysis for `python` if there won't be more Python files in the near future!

```suggestion
        language: [ 'cpp' ]
```

---

It's just for testing, so I think it's okay to remove it.
@justinclift What do you think? 😄

---

Hmmm, maybe lets see how much time is saved by removing it?

If it's not much, then we could leave it in.  That'd mean we can add more tests using Python at some point.

If leaving the Python stuff in there does make a significant difference though, it's probably best to remove it.  Faster test results means people can update and adjust their PRs faster (eg faster feedback cycle), which is important. :smile:

---

According to GitHub Actions, Python took 2min, while C++ took 14min. Given that analysis are run asynchronously, removing Python won't affect at all the time it takes for the CodeQL check to complete :)

---

Yes, Security can't be too emphasize. I guess my thinking was short.
I'll also keep the Python inspection code as well. and also, it's looks like we're ready to merge now, so I merged now. Thanks for contribution! :)